dashboard/src/modules/components/QuisbyChartsComponent/ChartGallery.jsx

```diff
@@ -0,0 +1,87 @@
+import {
+  BarElement,
+  CategoryScale,
+  Chart as ChartJS,
+  Legend,
+  LinearScale,
+  Title,
+  Tooltip,
+} from "chart.js";
+import {
+  Card,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateVariant,
+  Gallery,
+  GalleryItem,
+} from "@patternfly/react-core";
+
+import { Bar } from "react-chartjs-2";
+import React from "react";
+import { useSelector } from "react-redux";
+
+ChartJS.register(
+  BarElement,
+  Title,
+  Tooltip,
+  Legend,
+  CategoryScale,
+  LinearScale
+);
+const EmptyStateExtraSmall = (props) => (
+  <EmptyState variant={EmptyStateVariant.xs}>
+    <div>{props.message}</div>
+    <EmptyStateBody>Benchmark type is currently unsupported!</EmptyStateBody>
+  </EmptyState>
+);
+const ChartGallery = () => {
+  const { chartData, unsupportedType } = useSelector(
+    (state) => state.quisbyChart
+  );
+  return (
+    <>
+      {unsupportedType ? (
+        <EmptyStateExtraSmall message={unsupportedType} />
+      ) : (
+        <>
+          {chartData && chartData.length > 0 && (
```

Comment:
If we can assume that `chartData` is an array (which I _hope_ we can...), then checking it for "false" is redundant -- the `length > 0` test guard against the "empty" case.